docs/ref/templates/builtins.txt

```diff
@@ -1054,8 +1054,11 @@ this example, the space around ``Hello`` won't be stripped::
 
 Outputs one of the syntax characters used to compose template tags.
 
-Since the template system has no concept of "escaping", to display one of the
-bits used in template tags, you must use the ``{% templatetag %}`` tag.
+The template system has no concept of “escaping” individual characters.
```

Comment:
It looks like you switched to curly quotes here. Switch back to normal quotes.

Reply:
Good catch! done.